curp/src/server/cmd_worker/mod.rs

```diff
@@ -174,6 +174,33 @@ async fn worker_as<C: Command, CE: CommandExecutor<C>, RC: RoleChange>(
             sp.lock().remove(&entry.propose_id);
             let _ig = ucp.lock().remove(&entry.propose_id);
             if shutdown_self {
+                if let Some(maybe_new_leader) = curp.pick_new_leader() {
+                    info!(
+                        "the old leader {} will shutdown, try to move leadership to {}",
+                        id, maybe_new_leader
+                    );
+                    if curp
+                        .handle_move_leader(maybe_new_leader)
+                        .unwrap_or_default()
+                    {
+                        if let Err(e) = curp
+                            .connects()
+                            .get(&maybe_new_leader)
+                            .unwrap_or_else(|| {
+                                unreachable!("connect to {} should exist", maybe_new_leader)
+                            })
+                            .try_be_leader_now(curp.cfg().wait_synced_timeout)
+                            .await
+                        {
+                            warn!(
+                                "{} send timeout now to {} failed: {:?}",
```

Comment:
The logging message should be `send try be leader now`

Reply:
Fixed